appveyor.yml

```diff
@@ -11,7 +11,7 @@ environment:
     CSL_SYSTEM_TAG: "win64"
     CACHE_DIR: "c:\\c"
     WORK_DIR: "c:\\w"
-    DCONFIG: "qanet_tns_wallet"
+    DCONFIG: "ci_wallet"
```

Comment:
As Domen points out for travis.yml we need DCONFIG to be prefixed with either `testnet` or `mainnet`.